### PR TITLE
Feature/csr 24 - Nationality, Immigration and Employment Restrictions

### PR DESCRIPTION
**Pull request by fantasticjamieburns**

### CODE REVIEW

**Comment by fantasticjamieburns** [comment]
This PR is dependant on https://github.com/TransformCore/CSR-Fast-Stream-backend/pull/8 being deplyed first.

File: test/features/bootstrap/TransformCore/Bundle/AppBundle/Features/Context/FeatureContext.php [hunk 1]
```diff
@@ -24,4 +24,27 @@ class FeatureContext extends WebApiContext implements Context, SnippetAcceptingC
...
+    public function iSendARequestWithJson($method, $url, PyStringNode $string)
+    {   
+        $this->iSetHeaderWithValue('Content-Type', 'application/json');
+        $this->iSendARequestWithBody($method, $url, $string);
+    }
+
+    /**
+     * @Given following users for each persona exist on system:
+     */
+    public function followingUsersForEachPersonaExistOnSystem(TableNode $table)
```
Comment (fantasticjamieburns) [inline test/features/bootstrap/TransformCore/Bundle/AppBundle/Features/Context/FeatureContext.php]: Do we know if this should be here?

Comment (eddiejaoude) [inline test/features/bootstrap/TransformCore/Bundle/AppBundle/Features/Context/FeatureContext.php]: @jaffamonkey please see above

Comment (jaffamonkey) [inline test/features/bootstrap/TransformCore/Bundle/AppBundle/Features/Context/FeatureContext.php]: It is not needed (iSendARequestWithJson).  That was hang-over prior to using the webapicontext extension.

**Comment by knasher** [comment]
Reviewed: No issues found

File: test/features/bootstrap/TransformCore/Bundle/AppBundle/Features/Context/FeatureContext.php (hunk 1 above)
Comment (eddiejaoude) [inline test/features/bootstrap/TransformCore/Bundle/AppBundle/Features/Context/FeatureContext.php]: @jaffamonkey Can you remove this next time you are in there please. Thanks

**Pull request merged**